resources/css/bem/input-container.less

```diff
@@ -3,7 +3,9 @@
 
 .input-container {
   --label-font-size: @font-size--normal;
-  --label-height: calc(var(--label-font-size) + var(--input-padding-base) + 5px);
+  --label-height: calc(
```

Comment:
I think this should be `font-size * 1.25 (the line-height) + padding base` tbh 🤔

Reply:
eh close enough 🤷